docs/mutations.md

```diff
@@ -269,13 +348,127 @@ print(f"Before: {mts.num_mutations} mutations.\n"
+resulting in the chain V->G->I, which is impossible under this model
+(the chain is possible under the {class}`.BLOSUM62MutationModel`, but unlikely).
+In practice this is more confusing than concerning,
+since it can only happen when the same site is mutated more than once,
+which affects only a small proportion of sites.
+
+The situation above occurred because the probability of mutating to ``I``
+was very different depending on the initial state.
+However, if derived alleles are chosen in a way *independent* of the previous state,
+then adding mutations at rate :math:`\\alpha`
+and then adding mutations at rate :math:`\\beta`
+is *equivalent* to adding mutations once, at rate :math:`\\alpha + \\beta`.
+A mutation model has this property if rows of its transition matrix are the same,
+and is called *state independent* (or, *parent independent*).
+For instance, mutations under the {class}`.JC69MutationModel`
+choose *any* derived allele with equal probability.
+This means that the *result* of a mutation can be chosen without knowing
+what the previous state at that site was ("state independence"),
+which implies that adding new mutations above a given mutation does not affect
+the probability of that mutation ocurring.
+However, this is only true if "any derived allele" includes the previous allele,
+i.e., if we include *silent mutations*.
+The default Jukes-Cantor model will mutate an ``A`` to either a ``C``, ``G``, or ``T``
+with equal probability:
+
+```{code-cell}
+jc69 = msprime.JC69MutationModel()
+jc69.transition_matrix
+```
+
+We can check that this does not produce any silent mutations:
+
+```{code-cell}
+rate = 1.5
+ts = msprime.sim_ancestry(5, random_seed=5, sequence_length=100)
+mts = msprime.sim_mutations(ts, rate=rate, model=jc69, random_seed=7)
+count_transitions(mts, jc69.alleles)
+```
+
+Although this model does not include silent mutations,
+applying it more than once could produce some -
+for instance, if one mutation with derived state ``C`` is inserted
+above an existing mutation with the same derived state.
+To make it state-independent, we have to choose the derived state
+uniformly from *all* nucleotides:
+
+```{code-cell}
+pi_jc69 = msprime.JC69MutationModel(state_independent=True)
+pi_jc69.transition_matrix
+```
+
+With this change, mutating twice with {class}`.JC69MutationModel`
+is *equivalent* to mutating once with rate equal to the sum of the rates.
+In other words, these two operations are equivalent:
+
+```{code-cell}
+alpha = 0.4 * rate
+beta = 0.6 * rate
+# mutate once:
+mts1 = msprime.sim_mutations(ts, rate=alpha + beta, model=pi_jc69, random_seed=7)
+print("Mutated once:")
+count_transitions(mts1, pi_jc69.alleles)
+# or, mutate twice:
+mts2 = msprime.sim_mutations(ts, rate=alpha, model=pi_jc69, random_seed=7)
+mts2 = msprime.sim_mutations(mts2, rate=beta, model=pi_jc69, random_seed=8)
+print("Mutated twice:")
+count_transitions(mts2, pi_jc69.alleles)
+```
+
+These are not identical, but similar.
```

Comment:
"because we used different {ref}`random seeds<sec_mutations_controlling_randomness>`" (or whatever it is).

Reply:
Changed to "These are very similar - any differences are due to statistical noise.", since hopefully people wouldn't expect them to be *identical*. (Alternatively, I could explain that even setting random seeds we can't expect putting down mutations twice to be the same as putting down mutations once.)